docs/ops/sequence/AUGRUCell_dev.md

```diff
@@ -0,0 +1,144 @@
+# AUGRUCell  {#openvino_docs_ops_sequence_AUGRUCell_dev}
+
+**Versioned name**: *AUAUGRUCell-dev*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUCell* represents a single AUGRU Cell (GRU with attentional update gate).
+
+**Detailed description**: The main difference between *AUGRUCell* and [GRUCell](./GRUCell_3.md) is the additional attention score input `A`, which is a multiplier for the update gate.
```

Comment:
Could you please describe idea of usage of `AUGRUCell` shortly? I think it makes sense to add scientific explanation here and after you can refer to the original paper of it.

Reply:
I would like to finalize the discussion about the interface, formula, shapes and attributes of the operation.
I suggest to add proposals for more detailed description as a contribution in a sepatare PR.

Reply:
Agree, let us do it so.